helm/hpcc/templates/dali.yaml

```diff
@@ -56,16 +56,17 @@ spec:
     spec:
       {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" $dali.name "type" "dali") | indent 6 }}
       serviceAccountName: "hpcc-default"
+      {{- $sashaPlanes := list }}
+      {{- if not ( $sashaServices.disabled ) -}}
+      {{- range $sashaName, $_sasha := $dali.services -}}
+      {{- $sasha := ($_sasha | default dict) -}}
+      {{- if and (not $sasha.disabled ) ( $sasha.storage) }}
+        {{- $_ := append $sashaPlanes $sasha.storage.plane }}
```

Comment:
other blocks above could do with indenting too

Reply:
perhaps now that sasha storage can only point to a plane, we should get rid of "storage", and just have a single non-nested "storagePlane" ?

Reply:
There are two more changes that need to happen.  Changing labels, and working out the best way to define the default planes, and planes for a component.  E.g. storagePlane can be ambiguous.  Does it mean dataPlane, spillPlane, dllPlane?  Most of the time there is only one, but not always.